src/SparkTestBase.sol

```diff
@@ -96,8 +96,8 @@ abstract contract SparkTestBase is ProtocolV3TestBase {
             e2eTest(pool);
         }
 
-        GovHelpers.executePayload(vm, payload, executor);
-        
+        // GovHelpers.executePayload(vm, payload, executor);
```

Comment:
Why is this commented out?

Reply:
Ah sorry missed this after debugging GovHelper issues, added back in and tests still pass.